flink-runtime/src/main/java/org/apache/flink/runtime/memory/UnsafeMemoryBudget.java

```diff
@@ -36,16 +36,20 @@
  */
 class UnsafeMemoryBudget {
 	private static final int MAX_SLEEPS = 11; // 2^11 - 1 = (2 x 1024) - 1 ms ~ 2 s total sleep duration
-	private static final int MAX_SLEEPS_VERIFY_EMPTY = 17; // 2^17 - 1 = (128 x 1024) - 1 ms ~ 2 min total sleep duration
+	static final int MAX_SLEEPS_VERIFY_EMPTY_FOR_TESTS = 10; // 2^10 - 1 = (1 x 1024) - 1 ms ~ 1 s total sleep duration
```

Comment:
Testing code should not be contained in production code.

Reply:
True, I can move to the testing builder